tests/sentry/issues/test_ingest_incident_integration.py

```diff
@@ -169,3 +169,61 @@ def test_resolving_group_updates_incident(self) -> None:
         incident = Incident.objects.get(id=item.incident_id)
         activity = IncidentActivity.objects.filter(incident_id=incident.id)
         assert len(activity) == 4  # detected, created, priority change, close
+
+    def test_can_update_outstanding_incident_priority(self) -> None:
+        """Test that we link and update an incident that was opened before the org started single processing"""
+        _, group_info = self.save_issue_occurrence()
+        group = group_info.group
+        assert group is not None
+
+        assert GroupOpenPeriod.objects.filter(group=group, project=self.project).exists()
+
+        open_period = GroupOpenPeriod.objects.get(group=group, project=self.project)
+
+        dummy_relationship = IncidentGroupOpenPeriod.objects.get(group_open_period=open_period)
+        incident = Incident.objects.get(id=dummy_relationship.incident_id)
+
+        # this is a little hacky, but to emulate dual processing -> single processing I will
+        # just delete the IGOP relationship
+        IncidentGroupOpenPeriod.objects.all().delete()
+
+        _, group_info = self.save_issue_occurrence(priority=DetectorPriorityLevel.MEDIUM)
+        group = group_info.group
+        assert group is not None
+
+        item = IncidentGroupOpenPeriod.objects.get(group_open_period=open_period)
+        assert item.incident_id == incident.id
+        activity = IncidentActivity.objects.filter(incident_id=incident.id)
+
+        assert len(activity) == 4
+        last_activity_entry = activity[3]
+        assert last_activity_entry.type == 2
+        assert last_activity_entry.value == str(IncidentStatus.WARNING.value)
+        assert last_activity_entry.previous_value == str(IncidentStatus.CRITICAL.value)
+
+    def test_can_resolve_outstanding_incident(self) -> None:
+        """Test that we link and update an incident that was opened before the org started single processing on resolution"""
+        _, group_info = self.save_issue_occurrence()
+        group = group_info.group
+        assert group is not None
+
+        assert GroupOpenPeriod.objects.filter(group=group, project=self.project).exists()
+
+        open_period = GroupOpenPeriod.objects.get(group=group, project=self.project)
+
+        dummy_relationship = IncidentGroupOpenPeriod.objects.get(group_open_period=open_period)
+        incident = Incident.objects.get(id=dummy_relationship.incident_id)
+
+        # this is a little hacky, but to emulate dual processing -> single processing I will
+        # just delete the IGOP relationship
+        IncidentGroupOpenPeriod.objects.all().delete()
```

Comment:
nit but you could put lines 206-219 into a function to reuse in both tests since it's the same code, something like `emulate_dual_processing_to_single_processing` (or you can think of a better name)